### ISSUE TITLE
Use single color table for animated rotating flags

### ISSUE THREAD

**Issue by SomeAspy** [HIHIQY1/pride-overlay#13]
I couldn't help but notice that animated gifs made by this tool can be unreasonably large.
Using a single color table for all frames can (at times) dramatically reduce the size. (~30% in my cases)
if this method is applied during creation and not retroactively, this could also improve animation times.

since you are not supporting gifs (at least not yet) it may be worth applying this option compression method to all gifs. I'm not well versed in gif creation/editing in JS/TS so I can't offer much advice other than that.
![image](https://github.com/hexaclue/pride-overlay/assets/33640860/513379c1-809b-4f0c-a614-ab15ff3b19df)

**Comment by hexaclue** [comment]
True! The lib that's currently used (gifshot) has been taken down as well for whatever reason, so porting the gif exporting to a different library is probably a good idea. Haven't looked a lot into it yet, but [gif.js](https://github.com/jnordberg/gif.js) looks like a good contender.

Will look further into it when I have more time
